public/app/features/dashboard/utils/tracking.ts

```diff
@@ -0,0 +1,24 @@
+import { reportInteraction } from '@grafana/runtime';
+
+import { DashboardModel } from '../state';
+
+export function trackDashboardLoaded(dashboard: DashboardModel) {
+  // Count the different types of variables
+  const variables = dashboard.templating.list
+    .map((v) => v.type)
+    .reduce((r, k) => {
+      r[variableName(k)] = 1 + r[variableName(k)] || 1;
+      return r;
+    }, {});
+
+  reportInteraction('dashboard_loaded', {
+    uid: dashboard.uid,
+    title: dashboard.title,
+    style: dashboard.style,
```

Comment:
This `style` property is confusing to me 🤔 What information does it include?

Reply:
That's a good point. I'll change it to theme.